ld/nodemcu.ld

```diff
@@ -103,7 +103,11 @@ SECTIONS
     *(.init.literal)
     *(.init)
     
-    *sdk/esp_iot_sdk_*lib/lib*.a:*(.literal .text)
+    *libmain.a:*(.literal .text)
+    *libnet80211.a:*(.literal .text)
+    *libphy.a:*(.literal .text)
+    *libpp.a:*(.literal .text)
+    *libgcc.a:*(.literal .text)
```

Comment:
I'm *really* wary of this change altogether. As you already pointed out Yury, Espressif has put almost everything into `irom0.text`, except for what appears to be a few internal pieces. Presumably they had a really good reason for keeping those in IRAM. Maybe it would be worthwhile raising the large iram1_0_seg size with Espressif as a potential bug? If they respond, we'd know for certain whether it has to be kept in IRAM or not.